include/odp/api/spec/packet_io_stats.h

```diff
@@ -111,6 +126,69 @@ int odp_pktio_stats(odp_pktio_t pktio, odp_pktio_stats_t *stats);
  */
 int odp_pktio_stats_reset(odp_pktio_t pktio);
 
+/**
+ * Get extra statistics counter information for pktio handle
```

Comment:
... information for a packet IO interface

Spec text is more readable when the target of an operation is the interface, rather than the handle pointing to the interface.

Reply:
Fixed in V3.